lib/workers/branch/execute-post-upgrade-commands.ts

```diff
@@ -1,3 +1,4 @@
+import fs from 'fs';
```

Comment:
Nope, use our fs utils.
```suggestion
import fs from './../util/fs';
```